examples/basic_file.js

```diff
@@ -0,0 +1,37 @@
+import { config } from "dotenv";
+import { Acuvity, discoverApex } from "@acuvity/acuvity";
+
+import { dirname, resolve } from "path";
+import { fileURLToPath } from "url";
+
+const SCRIPT_DIR = dirname(fileURLToPath(import.meta.url));
+
+const filePath = resolve(SCRIPT_DIR, "test_data", "pi-test.txt");
+const configPath = resolve(SCRIPT_DIR, "configs", "simple_default_guard_config.yaml");
+
+console.log("File Path:", filePath);
+console.log("Config Path:", configPath);
+
+config();
+console.log("token", process.env.ACUVITY_TOKEN);
```

Comment:
I guess it's better for these examples if we simply fail when we don't have a token instead of logging it.

Reply:
anyway they will fail, let me rm the log.